rpc/wrtc_peer.go

```diff
@@ -145,28 +145,28 @@ func newPeerConnectionForServer(
 	config webrtc.Configuration,
 	disableTrickle bool,
 	logger golog.Logger,
-) (pc *webrtc.PeerConnection, dc *webrtc.DataChannel, err error) {
```

Comment:
As an aside, I have only found named returns to be hurtful/not helpful; almost always glad to move away from them 🧑‍🔧 .